### PR TITLE
Create GitHub Actions CI

### PR DESCRIPTION
**Pull request by ltoenning**
Resolves multiple aspects of #1.

### CODE REVIEW

File: .github/workflows/build-sola.yml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      run: |
+        mkdir build && cd build
+        cmake ../daisi -G Ninja -DCMAKE_BUILD_TYPE=Release -Dns3_DIR=~/ns-3_install/lib/cmake/ns3 -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
+        cmake --build .
+    - name: Run MINHTON unittest
+      run: build/sola_mono/minhton/tests/MinhtonAllTests
+    - name: Run Natter unittest
+      run: build/sola_mono/natter/tests/NatterAllTests
+    - name: Run CPPS unittest
+      run: build/tests/unittests/CPPSAmrMobilityHelperTest
```
Comment (janagoe) [inline .github/workflows/build-sola.yml:134]: add running unittests for...
DaisiDatastructureDirectedGraphTest
DaisiDatastructureWeightedDirectedGraphTest
DaisiDatastructureSimpleTemporalNetworkTest
DaisiCppsOrderManagementStnOrderManagement

or should we add something like CppsAllTests?

**Review by janagoe — APPROVED** [review]
Other than Cpps UnitTests in the Pipeline, everything looks fine 👍

File: .github/workflows/build-sola.yml (hunk 1 above)
Comment (ltoenning) [reply .github/workflows/build-sola.yml]: For now, I added all **DAISI** unittests individually.

**Review by janagoe — APPROVED** [review]
In the future, adding each test to the CI might become cumbersome, but its enough for now :)

**Pull request merged**